Orange/widgets/unsupervised/owmds.py

```diff
@@ -103,7 +103,7 @@ class Outputs:
 
     settingsHandler = settings.DomainContextHandler()
 
-    max_iter = settings.Setting(300)
+    max_iter = settings.Setting(10)
```

Comment:
What is this change doing in a commit "save selection in workflow" ?!?

Did you change this while testing and accidentally committed it?
Or decided to change some defaults and make it less obvious by not mentioning it at all and/or putting it in its own commit.